mapadroid/db/DbPogoProtoSubmit.py

```diff
@@ -199,7 +199,6 @@ def nearby_mons(self, origin: str, timestamp: float, map_proto: dict, mitm_mappe
 
                 """
                 if self._args.nearby_spawn_matching:
-                    #possible_spawns = self._db_exec.execute(query_spawns.format(lat=lat, lon=lon))
```

Comment:
Why delete that line, its within a full block of nearby scan rest

Reply:
Because Flake reported that line as an error and shows this PR as failing to build because of it.  The error is:
`mapadroid/db/DbPogoProtoSubmit.py:202:1: E800 Found commented out code`

Reply:
You can remove that whole block of commented code if you want

Reply:
Sure thing, I wasn't sure if anyone wanted it for future reference, but it looks cleaner without it.  I've removed it.